cmd/runtimetest/main.go

```diff
@@ -394,7 +450,7 @@ func validate(context *cli.Context) error {
 
 	linuxValidations := []validation{
 		validateDefaultFS,
-		validateDefaultDevices,
```

Comment:
We want to keep this (as long as the spec requires runtimes to supply those devices).